velodyne_pointcloud/src/lib/rawdata.cpp

```diff
@@ -266,10 +324,6 @@ void RawData::unpack(
       tmp.bytes[0] = block.data[k];
       tmp.bytes[1] = block.data[k + 1];
 
-      if (tmp.bytes[0] == 0 && tmp.bytes[1] == 0) {  // no laser beam return
-        continue;
-      }
-
```

Comment:
Why was this removed?

Reply:
Got to check that, but it looks like a merge mistake. So i will revert that change